sdk/communication/azure-communication-chat/azure/communication/chat/_shared/response.py

```diff
@@ -0,0 +1,8 @@
+# ------------------------------------------------------------------------
```

Comment:
is this chat package specific? if yes we should keep this out of  `_shared`  folder.  `_shared` is meant to be a common code to be used by all packages.

Reply:
This has been cleaned up.